apps/codeforafrica/src/components/RichText/RichText.js

```diff
@@ -91,13 +91,13 @@ const serialize = (children, props) =>
   });
 
 const RichText = React.forwardRef(function RichText(props, ref) {
-  const { color, elements, sx, ...other } = props;
+  const { color, elements, sx, typographyProps, ...other } = props;
```

Comment:
```suggestion
  const { color, elements, sx, typographyProps } = props;
```

1. We should pass only `typgraphyProps` to `serialize`.
2. If we need to handle `other`, then `other` should be applied to the root element i.e. `Box`.